src/components/SearchFilters.vue

```diff
@@ -18,9 +18,28 @@
           @tags-changed="tagsChangedCallback"
         ></custom-cascader>
         <div v-if="showFiltersText" class="filter-default-value">
-          <map-svg-icon icon="noun-filter" class="filter-icon-inside" />Apply
           Filters
         </div>
+        <el-popover
+          title="How do filters work?"
+          width="250"
+          trigger="click"
```

Comment:
Trigger on click not hover?

Reply:
@alan-wu would you prefer it on hover? I haven't gotten any design input on this one. Thought it might get in the way on hover since it does cover a few elements